Makefile

```diff
@@ -14,6 +14,15 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 #
 
+prefix ?= /usr/local
+exec_prefix ?= $(prefix)
+libexecdir ?= $(exec_prefix)/libexec
+# do not prefix with $(prefix) as these dirs are defined by the browsers
```

Comment:
Distros cannot influence these while packaging the browsers?

Reply:
These paths are distro specific, but independent of the prefix. If you install locally, you definitely want to install in `/usr/local`, but the browser paths are in `/usr` or `/etc` and by that independent of the prefix.

Reply:
Then the comment is misleading: These paths are defined by the distro packages of the browsers.